### ISSUE TITLE
Run WMCore central services validation for HG2111

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#10824]
**Impact of the new feature**
WMCore central services

**Is your feature request related to a problem? Please describe.**
Monthly task

**Describe the solution you'd like**
Validate central services in cmsweb-testbed (well, it might have to be in one of our VMs due to the current Rucio setup) and provide the final feedback by the November deadline specified by the CMSWEB team.

It also includes the creation of the service release notes and the validation check-list twiki.

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by amaltaro** [comment]
@todor-ivanov Todor, would you mind running this validation as well? As mentioned in the meeting yesterday, we might still have a few developments in by Thursday morning, but those should either be well isolated, or well validated from my side :) So please start it at your earliest convenience.

**Comment by todor-ivanov** [comment]
no problem @amaltaro

**Comment by todor-ivanov** [comment]
This month we decided to do this validation process completely under the K8 environment. We were planing to use the `dev` clusters for that. But since we had to configure the clusters to be used for this validation to point to the Rucio production instance and also because the `dev` environment is not considered fully isolated to a single person's activities (ergo prone to errors), we decided to switch and do the change in the `cmsweb-testbed` cluster [1]. We are keeping this configuration there for the time being.  

Here are the results from the validation process: [2]

[1]
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/120/diffs

[2]
https://twiki.cern.ch/twiki/bin/view/CMS/WMAgentEndtoEndValidationHG2111
